kDrive/UI/View/Files/Preview/VideoCollectionViewCell.swift

```diff
@@ -123,6 +137,25 @@ class VideoCollectionViewCell: PreviewCollectionViewCell {
         floatingPanelController?.dismiss(animated: true)
         parentViewController?.present(navController, animated: true) {
             playerViewController.player?.play()
+            let interval = CMTime(seconds: 1.0, preferredTimescale: CMTimeScale(NSEC_PER_SEC))
+            player.addPeriodicTimeObserver(forInterval: interval, queue: .main) { [weak self] _ in
+                self?.setNowPlayingMetadata()
+            }
+        }
```

Comment:
In the `SingleTrackPlayer` we don't call `setNowPlayingMetadata` to update time. Is it really needed ?